exec/java-exec/pom.xml

```diff
@@ -589,6 +669,17 @@
 
   <build>
 
+    <extensions>
+      <!--
+      Include the os-maven-plugin to get os.detected.classifier
+      -->
+      <extension>
+        <groupId>kr.motd.maven</groupId>
+        <artifactId>os-maven-plugin</artifactId>
+        <version>1.4.0.Final</version>
```

Comment:
Could not build Drill project on centos. Failed with exception:
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:2.4.1:shade (default) on project drill-jdbc-all: Error creating shaded jar: The name "os.detected.release.like."centos"" is not legal for JDOM/XML elements: XML names cannot contain the character """. -> [Help 1]`

It seems we need to use `1.5.0.Final` version (example of similar problem - https://github.com/apache/beam/pull/2391).